Remove duplicates of Acid1

gsnedders opened the pull request:
#### 590ad19dadef48cc5691fda64047c2d541fc892c
<pre>
Remove duplicates of Acid1
<a href="https://bugs.webkit.org/show_bug.cgi?id=270223">https://bugs.webkit.org/show_bug.cgi?id=270223</a>

Reviewed by Alan Baradlay.

This removes two of the three copies of Acid1 we have in our tree,
leaving only the version imported from the CSS 2.1 test suite:
css2.1/t09-c5526c-display-00-e.html (and this too will ultimately be
replaced when we finally import CSS2 from WPT, where Acid1 survives as
css/CSS2/css1/c5526c-display-000.xht).

* LayoutTests/css1/box_properties/acid_test.html: Removed.
* LayoutTests/fast/block/basic/011.html: Removed.
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/275549@main">https://commits.webkit.org/275549@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/da2f591a3a002d36208ef99c0d115c9eb1ab7d3b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/41847 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/20861 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/44229 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/44429 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/37942 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/24017 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/18192 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/34585 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/42421 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/17790 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/36037 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/15255 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/15478 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/37234 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/45825 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/38034 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/37381 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/41137 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/16662 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/13680 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/39584 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/18281 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/9436 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/18340 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/17925 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

gsnedders commented on LayoutTests/platform/mac/TestExpectations:
```diff
@@ -2530,8 +2530,6 @@ webkit.org/b/264177 requestidlecallback/requestidlecallback-deadline-shortened-b
 
 webkit.org/b/264527 imported/w3c/web-platform-tests/css/css-cascade/idlharness.html [ Pass Failure ]
 
-webkit.org/b/264607 [ Sonoma+ ] css1/box_properties/acid_test.html [ Pass Timeout ]
-
```
note that neither that bug (https://bugs.webkit.org/show_bug.cgi?id=264607) nor the results database (https://results.webkit.org/?suite=layout-tests&suite=layout-tests&suite=layout-tests&test=css1%2Fbox_properties%2Facid_test.html&test=css2.1%2Ft09-c5526c-display-00-e.html&test=fast%2Fblock%2Fbasic%2F011.html&platform=mac) show any pass/timeout flakiness—there's some FAIL a while ago, but that was resolved by rebaselining in https://github.com/WebKit/WebKit/commit/8849660df1a921225fff49a437e79b9c7fd04ebd, thus let's just remove this line entirely

Smackteo replied:
Reasonable but if the test is being removed in its entirely then we should do `find-expectations <test name>` to remove all expectations for each test we remove. (acid_test, and 011)

Which only reveals one expectation so this is actually all good.

gsnedders replied:
Yeah, I'd checked there weren't any other references. (check-webkit-style had caught the above!)

gsnedders committed:
Remove duplicates of Acid1
https://bugs.webkit.org/show_bug.cgi?id=270223

Reviewed by Alan Baradlay.

This removes two of the three copies of Acid1 we have in our tree,
leaving only the version imported from the CSS 2.1 test suite:
css2.1/t09-c5526c-display-00-e.html (and this too will ultimately be
replaced when we finally import CSS2 from WPT, where Acid1 survives as
css/CSS2/css1/c5526c-display-000.xht).

* LayoutTests/css1/box_properties/acid_test.html: Removed.
* LayoutTests/fast/block/basic/011.html: Removed.
* LayoutTests/platform/mac/TestExpectations:

Canonical link: https://commits.webkit.org/275549@main
